src/proteomes/components/entry/Publications.tsx

```diff
@@ -1,28 +1,36 @@
-import { FC } from 'react';
+import { FC, useMemo } from 'react';
 import { DataList } from 'franklin-sites';
 
-import CitationCard from '../../../supporting-data/citations/components/results/CitationsCard';
+import CitationsCard from '../../../supporting-data/citations/components/results/CitationsCard';
 
-import { getCitationItemId } from '../../../supporting-data/citations/utils';
-
-import { Citation } from '../../../supporting-data/citations/adapters/citationsConverter';
+import { CitationsAPIModel } from '../../../supporting-data/citations/adapters/citationsConverter';
 import { ProteomesAPIModel } from '../../adapters/proteomesConverter';
 
+import { getIdKeyFor } from '../../../shared/utils/getIdKeyForNamespace';
+import { Namespace } from '../../../shared/types/namespaces';
+
+const getIdKey = getIdKeyFor(Namespace.citations);
+
 const Publications: FC<Pick<ProteomesAPIModel, 'citations'>> = ({
   citations,
 }) => {
-  if (!citations?.length) {
+  const data = useMemo<CitationsAPIModel[]>(
+    () => citations?.map((citation) => ({ citation })),
```

Comment:
Could you please add a comment in the code to explain why this step is needed?

Reply:
Added a comment